cola/views.py

```diff
@@ -102,35 +96,38 @@ def get(self, request: HttpRequest, **kwargs: dict) -> HttpResponse:
         ):
             LOGGER.error("No cookie regex match found")
             return HttpResponse("Unauthorized", status=401)
-        jwk_response = get_cola_cognito_user_pool_jwk()
-        if not jwk_response:
-            return HttpResponseServerError()  # TODO: Improve error handling throughout
-        else:
-            cola_cognito_user_pool_jwk, cola_issuer = jwk_response
+
+        response = requests.get(COLA_URL, timeout=5)
+        if response.status_code != 200:
+            LOGGER.error("failed to get expected response from COLA")
+            return HttpResponse("Unauthorized", status=401)
+        cola_cognito_user_pool_jwk = response.json()
```

Comment:
I returned this to how you had it originally @252afh, which i think is better, after i found a better way to mock it